variables.tf

```diff
@@ -3,9 +3,15 @@ variable "bucket" {
   type        = string
 }
 
-variable "use_account_alias_prefix" {
```

Comment:
What do you think about keeping the `use_account_alias_prefix` variable?  That way folks already using this module don't need make any modifications.